src/js/components/hocs.js

```diff
@@ -51,19 +51,19 @@ export const withFocus = (WrappedComponent) => {
     }
 
     handleActiveMouse = () => {
-      this.mouseActive = true;
+      window.mouseActive = true;
```

Comment:
Why does this need to be window?

Reply:
`Attaching mouseActive to the window object to avoid problems with portals`

Reply:
mouseActive is getting set to false when opening a portal because `this.mouseActive` in a new portal is always false.